### PR TITLE
Np 111

### PR DESCRIPTION
**Pull request by GaneshMaddipoti**
first test to call uknw auth checker api-200-ok

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/api/service/AuthService.scala [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import uk.gov.hmrc.api.client.HttpClient
+import uk.gov.hmrc.api.conf.TestConfiguration
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class AuthService extends HttpClient {
+  val host: String = TestConfiguration.url("auth")
+
+  val authPayload: String =
```
Comment (TheDanielDoyle) [inline src/test/scala/uk/gov/hmrc/api/service/AuthService.scala:18]: This payload is for another service. We need a payload for our service.

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -17,9 +17,17 @@ environment: local
...
   services {
     host: "http://localhost"
-    service-name-frontend {
-      port: 1234
-      productionRoute: "/path"
+    auth {
+      port: 8585
+      productionRoute: ""
+    }
+    ims {
```
Comment (TheDanielDoyle) [inline src/test/resources/application.conf:24]: This key looks unused?

File: src/test/resources/application.conf [hunk 3]
```diff
@@ -17,9 +17,17 @@ environment: local
 local {
   services {
     host: "http://localhost"
-    service-name-frontend {
-      port: 1234
-      productionRoute: "/path"
+    auth {
+      port: 8585
+      productionRoute: ""
```
Comment (TheDanielDoyle) [inline src/test/resources/application.conf:22]: What happens if you omit the empty keys?

File: src/test/resources/requests/authRequest200_single.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "date":"2024-02-08",
+  "authType": "UKNW",
+  "eoris" : ["GB0000000200"]
```
Comment (TheDanielDoyle) [inline src/test/resources/requests/authRequest200_single.json:4]: Please change to GB000000000200
GB0000000200 does not match the regex for EORis

File: src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+
+class UknwAuthCheckerApiService extends HttpClient {
+
+  val authorisationsUrl: String = TestConfiguration.url("uknw-auth-checker-api")
+
+  def authorisations(authToken: String): StandaloneWSRequest#Self#Response = {
+    val individualPayload = getJsonFile("authRequest200_single.json")
```
Comment (TheDanielDoyle) [inline src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala:17]: loading this here does not seem wise. Should be in the spec.

File: src/test/scala/uk/gov/hmrc/api/service/AuthService.scala (hunk 1 above)
Comment (GaneshMaddipoti) [reply src/test/scala/uk/gov/hmrc/api/service/AuthService.scala:29]: Replaced with proper payload

File: src/test/resources/application.conf (hunk 2 above)
Comment (GaneshMaddipoti) [reply src/test/resources/application.conf:24]: Removed IMS as we are not using.

File: src/test/resources/application.conf (hunk 3 above)
Comment (GaneshMaddipoti) [reply src/test/resources/application.conf:22]: Its been used in code to form the full URL so

File: src/test/resources/requests/authRequest200_single.json (hunk 4 above)
Comment (GaneshMaddipoti) [reply src/test/resources/requests/authRequest200_single.json:4]: Changed EORI

File: src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala (hunk 5 above)
Comment (GaneshMaddipoti) [reply src/test/scala/uk/gov/hmrc/api/service/UknwAuthCheckerApiService.scala]: Refactored code to send the the test data from Spec

**Review by TheDanielDoyle — DISMISSED** [review]

**Review by TheDanielDoyle — APPROVED** [review]

**Pull request merged**